app/components/work_packages/share/bulk_permission_button_component.html.erb

```diff
@@ -4,7 +4,7 @@
                                        anchor_align: :end,
                                        color: :subtle,
                                        data: { test_selector: 'op-share-wp-bulk-update-role'})) do |menu|
-    menu.with_show_button(data: { 'work-packages--share--bulk-selection-target': 'bulkUpdateRoleLabel' }) do |button|
+    menu.with_show_button(scheme: :invisible, color: :subtle, data: { 'work-packages--share--bulk-selection-target': 'bulkUpdateRoleLabel' }) do |button|
```

Comment:
I fiddled around a bunch trying to find the scheme that would make the buttons look like they do with this PR. Never thought of combining a scheme and a color. Thank you :)